blocks/network-link/network-link.php

```diff
@@ -0,0 +1,134 @@
+	$link .= '</span></a></li>';
+
+	return $link;
+}
+
+/**
+ * Returns the SVG for social link.
+ *
+ * @param string $service The service slug to extract data from.
+ * @param string $field The field ('name', 'icon', etc) to extract for a service.
+ * @return array|string
+ */
+function block_cata_network_link_services( string $service = '', string $field = '' ) : array|string {
+	$services_data = array(
+		'collectiveworld' => array(
+			'name' => 'Collective World',
+			'icon' => '<svg viewBox="0 0 200 20.05"><path d="M0 10.17C0 3.99 4.63.28 9.55.28s7.68 2.84 8.54 6.16l-3.02 1.01c-.56-2.3-2.33-4.15-5.51-4.15s-6.28 2.27-6.28 6.87 3.05 6.8 6.29 6.8 5-2.04 5.64-4.18l2.95.97c-.85 3.21-3.75 6.29-8.59 6.29-5.1 0-9.57-3.69-9.57-9.88Zm33.19 2.95c0 4.02-2.87 6.93-6.77 6.93s-6.74-2.92-6.74-6.93 2.87-6.9 6.74-6.9 6.77 2.91 6.77 6.9Zm-3.12 0c0-2.7-1.71-4.1-3.65-4.1s-3.65 1.39-3.65 4.1 1.74 4.15 3.65 4.15 3.65-1.42 3.65-4.15Zm5.58 6.52V.28h3.11v19.37h-3.11Zm6.58 0V.28h3.11v19.37h-3.11Zm18.13-3.72c-.7 2.27-2.75 4.12-5.89 4.12-3.53 0-6.67-2.58-6.67-6.99 0-4.12 3.05-6.86 6.34-6.86 4.02 0 6.37 2.65 6.37 6.77.01.33-.01.67-.07 1h-9.52c.07 1.98 1.64 3.4 3.56 3.4s2.84-1 3.27-2.27l2.62.82Zm-2.98-4.22c-.06-1.52-1.07-2.89-3.21-2.89-.8-.01-1.57.28-2.16.82-.59.54-.95 1.28-1.02 2.08h6.39Zm8.19 1.42c0 2.62 1.69 4.1 3.65 4.1s2.97-1.42 3.27-2.4l2.74.98c-.61 2.04-2.58 4.26-5.99 4.26-3.84 0-6.74-2.97-6.74-6.93s2.92-6.9 6.67-6.9c3.5 0 5.44 2.2 5.96 4.28l-2.78 1.01c-.29-1.16-1.17-2.43-3.15-2.43s-3.62 1.42-3.62 4.04ZM81.9 6.61h2.7v2.75h-2.7v6.05c0 1.16.51 1.64 1.67 1.64.36 0 .72-.04 1.07-.1v2.58c-.19.08-.78.29-1.89.29-2.42 0-3.91-1.45-3.91-3.88V9.39h-2.42V6.62h.67c1.39 0 2.01-.88 2.01-2.04V2.64h2.81v3.97ZM89.31 0a2.035 2.035 0 0 1 2.04 2.02 2.03 2.03 0 0 1-3.9.78c-.05-.12-.09-.25-.12-.38-.08-.39-.04-.8.11-1.17A2.031 2.031 0 0 1 89.3-.01Zm-1.52 19.65V6.61h3.08v13.04h-3.08Zm13.76 0h-3.11L93.15 6.61h3.43l3.46 9.32 3.37-9.32h3.27l-5.12 13.04Zm18.52-3.72c-.7 2.27-2.76 4.12-5.89 4.12-3.53 0-6.67-2.58-6.67-6.99 0-4.12 3.05-6.86 6.34-6.86 4.02 0 6.37 2.65 6.37 6.77.01.34-.01.67-.08 1h-9.51c.08 1.98 1.64 3.4 3.56 3.4s2.84-1 3.27-2.27l2.62.82Zm-2.97-4.22c-.06-1.52-1.07-2.89-3.21-2.89-.8-.01-1.57.28-2.17.82s-.96 1.28-1.02 2.08h6.39ZM151.91.68h3.31l-5.33 18.98h-3.31l-4.6-14.35-4.6 14.34h-3.3L128.7.68h3.37l3.81 13.86L140.32.68h3.34l4.53 13.98L151.92.68Zm16.65 12.44c0 4.02-2.87 6.93-6.77 6.93s-6.74-2.92-6.74-6.93 2.87-6.9 6.74-6.9 6.77 2.91 6.77 6.9Zm-3.13 0c0-2.7-1.71-4.1-3.65-4.1s-3.65 1.39-3.65 4.1 1.74 4.15 3.65 4.15 3.63-1.43 3.65-4.15Zm13.21-3.43c-.33-.05-.67-.08-1.01-.07-2.42 0-3.5 1.39-3.5 3.84v6.21h-3.11V6.63h3.02v2.09c.61-1.42 2.05-2.24 3.78-2.24.28 0 .55.03.82.08V9.7Zm2.61 9.96V.28h3.11v19.37h-3.11Zm18.62-2.37c0 .8.04 1.6.13 2.39h-2.97c-.09-.56-.13-1.13-.13-1.69-.62 1.1-1.98 2-3.82 2-3.75 0-6.27-2.95-6.27-6.87s2.55-6.83 6.21-6.83c2.27 0 3.43 1.04 3.84 1.88V.28h3.02l-.02 17Zm-6.44-.03c2.04 0 3.46-1.69 3.46-4.18s-1.39-4.04-3.43-4.04-3.53 1.58-3.53 4.07 1.37 4.15 3.5 4.15Z"/></svg>',
```

Comment:
I think the icons should have `width` and `height` attributes so that during page load they take up space and have their  correct aspect-ratio until the CSS is applied.

Reply:
Good point @douglas-johnson, this is updated and on the CW dev site